onnx_tf/handlers/backend/cast.py

```diff
@@ -20,3 +20,10 @@ def version_1(cls, node, **kwargs):
   @classmethod
   def version_6(cls, node, **kwargs):
     return [cls.make_tensor_from_onnx_node(node, **kwargs)]
+
+  @classmethod
+  def version_9(cls, node, **kwargs):
+    inp = kwargs["tensor_dict"][node.inputs[0]]
+    if (node.attrs.get("to") == tf.string):
```

Comment:
It seems like casting **from** string type would cause problem?
For instance, if we want to convert "100" to 100, this handler would err.

Reply:
@tjingrant good catch! Updated handler and test code to cover cast from strings. Please review.